shoop_tests/core/test_configurations.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2015, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+import pytest
+
+from shoop import configuration
+from shoop.core import cache
+from shoop.core.models import Configuration
+from shoop.testing.factories import get_default_shop
+
+
+@pytest.mark.django_db
+def test_configurations_1():
+    cache.clear()
+    shop = get_default_shop()
+    test_conf_data = {"data": "test"}
+    configuration.set(shop, "key", test_conf_data)
+
+    assert configuration.get(shop, "key") == test_conf_data
+    assert Configuration.objects.get(shop=shop, key="key").data == test_conf_data
+    assert cache.get(configuration.SHOP_CONFIGURATION_CACHE_PREFIX % shop.pk).get("key") == test_conf_data
```

Comment:
The fact that the configuration API is backed by the cache shouldn't be very public.

Of course you'll want to test that stuff gets cached (again, more descriptive test names, please?), but is it a good idea that all three tests touch the cache?